jicofo/src/main/kotlin/org/jitsi/jicofo/jibri/JibriRecorder.kt

```diff
@@ -108,10 +108,10 @@ class JibriRecorder(
                     jibriDetector,
                     false, null, iq.displayName, iq.streamId, iq.youtubeBroadcastId, sessionId, iq.appData,
                     logger
-                ).apply {
-                    start()
-                    logger.info("Started Jibri session")
-                }
+                )
+                this.jibriSession = jibriSession
```

Comment:
I see why you removed the `apply`, but why do you need the `val`?

Reply:
Otherwise I have to deal with `this.jibriSession` potentially being null